prompt_toolkit/shortcuts/progress_bar/formatters.py

```diff
@@ -198,6 +199,29 @@ def get_width(self, progress_bar: 'ProgressBar') -> AnyDimension:
         return D.exact(max(all_lengths) * 2 + 1)
 
 
+class Counter(Formatter):
+    """
+    Display the items competed as text.  E.g. "8"
```

Comment:
Can you add ``Counter`` to ``__all__`` at the top of this file as well?

Reply:
done